WORKSPACE

```diff
@@ -466,3 +466,12 @@ maven_jar(
   artifact = "com.yahoo.pulsar:pulsar-client:jar:shaded:1.18"
 )
 # end Pulsar Client
+
+# for pex rules
+git_repository(
+    name = "io_bazel_rules_pex",
+    remote = "https://github.com/streamlio/bazel_rules_pex.git",
```

Comment:
agreed - the repo can be easily duplicated since once it is deployed the changes will be none or minimal